Subject: Don't require libdb

+++ a/debian/control.in
@@ -7,7 +7,7 @@
   lsb-release, sharutils,
   libreadline-dev | libeditreadline-dev, libncursesw5-dev (>= 5.3), @bd_gcc@
   zlib1g-dev, libbz2-dev, liblzma-dev,
-  libgdbm-dev, @bd_dbm@,
+  libgdbm-dev,
   tk-dev, blt-dev (>= 2.4z), libssl-dev,
   libexpat1-dev, libmpdec-dev (>= 2.5.1~),
   libbluetooth-dev [linux-any] <!pkg.@PVER@.nobluetooth>,
+++ a/debian/rules
@@ -88,8 +88,7 @@
 # FIXME: remove sid when uploading to unstable, after bookworm is released
 #ifneq (,$(filter $(distrelase), jessie stretch buster bullseye bookworm sid precise trusty xenial bionic focal groovy hirsute))
   with_dbmmodule = yes
-  bd_dbm = libdb-dev
-  dbmliborder = bdb:gdbm
+  dbmliborder = gdbm
 #else
 #  bd_dbm = libgdbm-compat-dev
 #  dbmliborder = gdbm:bdb
@@ -812,7 +811,6 @@
 	    -e "s/@MINPRIO@/$(PY_MINPRIO)/g" \
 	    -e "s/@bd_qual@/$(bd_qual)/g" \
 	    -e "s/@bd_gcc@/$(bd_gcc)/g" \
-	    -e "s/@bd_dbm@/$(bd_dbm)/g" \
 	    -e "s/@bd_dpkgdev@/$(bd_dpkgdev)/g" \
 		debian/control.in \
 		$(if $(with_udeb),debian/control.udeb) \
